### PR TITLE
Add responsive behavior for main text and footer

### PR DESCRIPTION
**Pull request by borisyankov**
Moved from redesign fork to main project.

### CODE REVIEW

File: src/sass/media-queries.scss [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-                              }
-                          }
-                      }
-                  }
-              }
-          }
-      }
-  }
-
-  #sell_popup_container {
```
Comment (raunakkathuria) [inline src/sass/media-queries.scss]: have we fixed this somewhere else? It was added so that we show our sell popup as 100% on mobile and 7 -inch tablet

Comment (borisyankov) [inline src/sass/media-queries.scss]: Yeah, none of the styles are needed but I have forgotten to include the non-media-query #sell_popup_container style. Will redo the PR, but apart from that, these are not needed.

**Comment by raunakkathuria** [comment]
Can we instead of creating pull request directly to master create it for some other branch so that it can be tested on QA?

**Comment by borisyankov** [comment]
It is in a branch on the main repo, so the 'responsive' branch can be tested on QA, I think?

**Pull request closed without merging**